### DISCUSSION TITLE
Editable Dialog Box for Bricks in Areas

### DISCUSSION THREAD

**Discussion by SaschaHowe** [pimcore/pimcore#8207] [Q&A / Support] [answered]
Hey, 

#6923 introduced the EditableDialogBoxConfiguration for Bricks, is there any way to access the configuration box when the brick is not in an `Areablock` but just an `Area` that I miss? 

It currently looks like no, though I think it would be equally viable to be able to configure a brick in the context of an `Area`, too. 

Greetings, 
    Sascha

**Comment by dvesh3** [comment, marked as answer]
Thanks for reporting. Planned for v.6.9 https://github.com/pimcore/pimcore/issues/8240

**Reply by SaschaHowe** [reply]
awesome!

**Reply by passioneight** [reply]
Does this actually work for you on `6.9.6`? In my case the data set in the `EditableDialogBoxConfiguration` is only available if the `Brick` is in an `Areablock`, but not if it is rendered using `pimcore_area`.

So, the exact same `Brick` is behaving differently, depending on whether it was added in an `Areablock` (working) or rendered as `Area` (not working).
